pkgs/data/themes/marwaita/default.nix

```diff
@@ -8,7 +8,7 @@
 
 stdenv.mkDerivation rec {
   pname = "marwaita";
-  version = "9.1";
+  version = "9.2";
 
   src = fetchFromGitHub {
     owner = "darkomarko42";
```

Comment:
Missing hash update.

Reply:
Upstream just increased the version number, keeping the same source revision as of version 9.1

Reply:
https://github.com/darkomarko42/Marwaita/issues/11

Reply:
There is https://github.com/darkomarko42/Marwaita/compare/9.1...master

Reply:
Comparing 9.1 and 9.2 shows that they are the same: https://github.com/darkomarko42/Marwaita/compare/9.1...9.2
So my intent with this PR is just to use the latest version number.